build_tools/benchmarks/common/benchmark_driver.py

```diff
@@ -68,20 +68,24 @@ def run(self) -> None:
 
     use_legacy_name = self.benchmark_suite.legacy_suite
 
-    target_architectures = []
-    cpu_target_arch = self.device_info.get_iree_cpu_arch_name(use_legacy_name)
-    if cpu_target_arch is None:
-      print("WARNING: Detected unsupported CPU architecture in "
-            f'"{self.device_info}", CPU benchmarking is disabled.')
-    else:
-      target_architectures.append(cpu_target_arch)
+    if self.config.use_compatible_filter:
+      compatible_architectures = []
+      cpu_target_arch = self.device_info.get_iree_cpu_arch_name(use_legacy_name)
+      if cpu_target_arch is None:
+        print("WARNING: Detected unsupported CPU architecture in "
+              f'"{self.device_info}", CPU benchmarking is disabled.')
```

Comment:
Given that this is now only happening when a user passes a specific flag, I think we can downgrade to info. Expected warnings are annoying.



```suggestion
        print(f"INFO: Detected incompatible CPU archiecture architecture"
              f" '{self.device_info}'. Disabling CPU benchmarking because"
              f" --compatible-only was passed")
```

Reply:
SG. Done.